tests/test_interfaces.py

```diff
@@ -0,0 +1,779 @@
+# pylint: skip-file
+""" This module tests the interface module """
+
+import unittest
+import mock
+import wifiphisher.common.interfaces as interfaces
+import pyric
+
+
+class TestNetworkAdapter(unittest.TestCase):
+    """ Tests NetworkAdapter class """
+
+    @mock.patch("wifiphisher.common.interfaces.pyw")
```

Comment:
I believe it was because of `pyw.card`. However we don't need it and I will remove it.